bin/_help

```diff
@@ -7,34 +7,33 @@
 source "/usr/local/bin/_common"
 
 print_common_usage() {
-  echo "Usage: nib ${1} [SERVICE]"
+  echo "Usage: nib $COMMAND [SERVICE]"
```

Comment:
Magic variable `$COMMAND` is always set, no need to pass/eval argument in this method. Should this not be part of this PR?